src/pages/changelog.js

```diff
@@ -0,0 +1,11 @@
+import React, { useEffect } from 'react';
```

Comment:
the vercel redirect should work for all prod cases.

do we need a local/development redirect too? 

Meaning - this page only really matters when not hitting slugs at docs.temporal.io - right?